### ISSUE TITLE
Does Localization covers data annotations?

### ISSUE THREAD

**Issue by giannik** [OrchardCoreContrib/OrchardCoreContrib.Modules#62]
Hello @hishamco 
Does the localization module cover Data annotations placed on model properties for Validation.
Example  The Error Message  data annotation for validation ?
Is there an example on how to use?

**Comment by hishamco** [comment]
Nope, it's not support, the intend of this module to localize the dynamic strings, I'm not sure what's the case the require you to do in the data annotations

FYI we already implement data annotations localization in OC, I can refer you to the PR if you missed

**Comment by giannik** [comment]
yes please,
what pr is for data annotations localization ?

**Comment by hishamco** [comment]
https://github.com/OrchardCMS/OrchardCore/pull/4675
